WalletWasabi/Crypto/Attribute.cs

```diff
@@ -0,0 +1,21 @@
+using NBitcoin;
+using NBitcoin.Secp256k1;
+using WalletWasabi.Crypto.Groups;
+
+namespace WalletWasabi.Crypto
+{
+	public class Attribute
```

Comment:
Maybe rename to `UnsignedCredential` because it parallels `Credential`? This name kind of implies a credential has an attribute. `CredentialRequest` also kind of makes sense, but since this contains the secret randomness as a property I don't think it would be appropriate.

Reply:
This class makes no sense. I will remove it.